apps/pipelines/nodes/helpers.py

```diff
@@ -55,10 +55,26 @@ def _get_db_object(self):
             )
         return self._participant_data
 
-    def get(self):
-        return self._get_db_object().data
+    def get(self, include_global=True):
+        data = self._get_db_object().data
+        if include_global and hasattr(self.session, "participant") and hasattr(self.session.participant, "global_data"):
```

Comment:
I don't think these checks are necessary, a session will always have a participant and a participant will always have the 'global_data' property